com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/ILifeCycleEventListener.kt

```diff
@@ -0,0 +1,9 @@
+package com.chartboost.heliumsdk.unity
+
+/**
+ * Interface for sending real-time lifecycle events
+ */
+interface ILifeCycleEventListener {
+    fun DidStart(errorCode: Int, errorDescription: String)
+    fun DidReceiveILRD(impressionDataJson: String)
```

Comment:
Function names should start with a lowercase letter: https://kotlinlang.org/docs/coding-conventions.html#function-names.

Reply:
Look feedback above.